recipes/cfanalytics/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "cfanalytics" %}
+{% set version = "0.1.4" %}
+{% set sha256 = "aa3d98ebc1c39000c1e7ab08d586e079f8d7c87db15a318e03ff4918bc38c739" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py<36]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
```

Comment:
You could set the python version requirement here, then this could be a `noarch` package.

Reply:
(you most likely also need the >=3.6 here too)